src/main/java/caspar/spaceport/industries/com/destinations/model/Destination.java

```diff
@@ -0,0 +1,21 @@
+package caspar.spaceport.industries.com.destinations.model;
+
+public class Destination {
```

Comment:
Could this be changed to a record instead of a POJO? (If the fields will continue to be immutable)